tox.ini

```diff
@@ -175,7 +173,8 @@ commands =
     python -m towncrier --draft --config towncrier.toml
 
 [testenv:news]
-passenv = TAHOE_LAFS_* PIP_* SUBUNITREPORTER_* USERPROFILE HOMEDRIVE HOMEPATH
+# On macOS, git invoked from Tox needs $HOME.
+passenv = HOME * TAHOE_LAFS_* PIP_* SUBUNITREPORTER_* USERPROFILE HOMEDRIVE HOMEPATH
```

Comment:
Looks like there's a stray `*` there?

Reply:
Silly of me, I assumed `*` was the separator.  Thank you for catching that!

Reply:
Dropped the `*` in 2494980.